.travis.yml

```diff
@@ -1,11 +1,6 @@
-language: go
-dist: xenial
+dist: focal
 os: linux
 
-go:
-  - "1.15"
```

Comment:
We actually rely on this as part of our test matrix - it gets set as TRAVIS_GO_VERSION, which gets referenced in docker-compose.yml. It's true that Travis doesn't need to actually install Go, but this is a convenient way to test 2 versions of Go at once, which we frequently need during transitions.

I'd like to either keep the go stanza, or do the same thing with a Travis `matrix` and leave around the one current version of Go we use.

Reply:
Oh gosh, this is a fantastic catch! I'll get the versions added back in and omit the language line and see if we can still avoid the installs.